src/script/page/MainContent/panels/Collection/Collection.test.tsx

```diff
@@ -85,10 +89,18 @@ describe('Collection', () => {
   const mockConversationRepository = {
     getEventsForCategory: jest.fn().mockResolvedValue(messages),
     searchInConversation: jest.fn().mockResolvedValue({messageEntities: [createLinkMessage()], query: 'term'}),
-  };
+  } as Partial<ConversationRepository>;
```

Comment:
const mockConversationRepository: Partial<ConversationRepository> = {}
 
 would be the preferable way of typing here.